main.py

```diff
@@ -1,10 +1,59 @@
 import init_django_orm  # noqa: F401
+import json
+
 
 from db.models import Race, Skill, Player, Guild
 
 
+def create_guild(data: dict):
+    if 'description' in data :
```

Comment:
You don't need this condition. You can use the dict method `get` with the default value `None`, because guild description can be `null`.